src/index.html

```diff
@@ -11,7 +11,212 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      onsubmit="onSubmit()"
+    >
+      <fieldset class="fieldset_item">
+        <legend>Personal information</legend>
+        <div class="form_item">
+          <label for="surname">
+            Surname:
+            <input
+              id="surname"
+              type="text"
+              name="name"
+              required
+            >
+          </label>
+        </div>
+
+        <div class="form_item">
+          <label for="name">
+            Name:
+            <input
+              type="text"
+              name="name"
+              id="name"
+              required
+            >
+          </label>
+        </div>
+
+
+        <div class="form_item">
+          <label for="old">
```

Comment:
one blank line maximum

```suggestion
          </label>
        </div>

        <div class="form_item">
          <label for="old">
```